setup.py

```diff
@@ -52,7 +52,7 @@
 DOWNLOAD_URL = "https://github.com/pydicom/pydicom/archive/master.zip"
 LICENSE = "MIT"
 VERSION = __version__
-# PACKAGE_DATA = {'pydicom': ['../tests/tests_files/*']}
+PACKAGE_DATA = {'pydicom': ['tests/test_files/*', 'tests/charset_files/*']}
 REQUIRES = []
```

Comment:
I don't think that the audience as anything to do with that. We just want to be able to test against an installed version which is a good thing (which I think refer to what you call validation). We could actually to that by keeping the tests outside. But finding the tests belonging to a specific module is much less intuitive than the inside test scheme. So to combine both pro (inside test and be able for the validation) you need to copy the file at install.

And pytest allow you to easily test against an installed version when you test are inside using `py.test --pyargs mypackg` which is more difficult to do if they are outside and installed from pypi.